packages/sdk/src/types/Orders.ts

```diff
@@ -0,0 +1,75 @@
+export interface OrdersItem {
+	sku_id?: string
+	product_id?: string
+	related_object?: 'product' | 'sku'
+	source_id?: string
+	quantity?: number
+	price?: number
+	amount?: number
+	product?: {
+		override?: boolean
+		name?: string
+		metadata?: Record<string, any>
+	}
+	sku?: {
+		override?: boolean
+		sku?: string
+	}
+}
+
+export interface OrdersCreate {
+	customer?: {
+		source_id?: string
+		name?: string
+		email?: string
+		metadata?: Record<string, any>
+	}
+	amount?: number
+	items?: OrdersItem[]
+	metadata?: Record<string, any>
+}
+
+export interface OrdersCreateResponse {
+	id?: string
+	source_id?: string
+	created_at?: string
```

Comment:
Please drop `?`

Reply:
Dropped only for `created_at`, or should it be dropped for `id` also?